content/hi/docs/reference/glossary/csi.md

```diff
@@ -0,0 +1,21 @@
+---
+title: कंटेनर स्टोरेज इंटरफ़ेस (CSI)
+id: csi
+date: 2018-06-25
+full_link: /docs/concepts/storage/volumes/#csi
+short_description: >
+    कंटेनर स्टोरेज इंटरफेस (CSI) स्टोरेज सिस्टम को कंटेनरों में एक्सपोज करने के लिए एक मानक इंटरफेस को परिभाषित करता है।
+
+
+aka: 
+tags:
+- storage 
+---
+ कंटेनर स्टोरेज इंटरफेस (CSI) स्टोरेज सिस्टम को कंटेनरों में एक्सपोज करने के लिए एक मानक इंटरफेस को परिभाषित करता है।
+
+<!--more--> 
+
+CSI विक्रेताओं को कुबेरनेट्स के लिए कस्टम स्टोरेज प्लगइन्स बनाने की अनुमति देते हैं, बिना किसी कुबेरनेट्स रिपॉजिटरी (आउट-ऑफ-ट्री प्लगइन्स) में शामिल हुएँ। स्टोरेज प्रदाता से CSI ड्राइवर का उपयोग करने के लिए, आपको पहले [इसे अपने क्लस्टर में डिप्लॉय करना होगा](https://kubernetes-csi.github.io/docs/deploying.html)। फिर आप एक {{< glossary_tooltip text="स्टोरेज क्लास" term_id="storage-class" >}} बना सकते हैं जो इस CSI ड्राइवर का उपयोग करेगा।
```

Comment:
I don't think किसी is required here since the original text doesn't have "any" in the sentence.
```suggestion
CSI विक्रेताओं को कुबेरनेट्स के लिए कस्टम स्टोरेज प्लगइन्स बनाने की अनुमति देते हैं, बिना कुबेरनेट्स रिपॉजिटरी (आउट-ऑफ-ट्री प्लगइन्स) में शामिल हुएँ। स्टोरेज प्रदाता से CSI ड्राइवर का उपयोग करने के लिए, आपको पहले [इसे अपने क्लस्टर में डिप्लॉय करना होगा](https://kubernetes-csi.github.io/docs/deploying.html)। फिर आप एक {{< glossary_tooltip text="स्टोरेज क्लास" term_id="storage-class" >}} बना सकते हैं जो इस CSI ड्राइवर का उपयोग करेगा।
```